cmd/experimental/kjobctl/pkg/builder/templates/slurm_init_entrypoint_script.sh.tmpl

```diff
@@ -0,0 +1,62 @@
+#!/usr/local/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
```

Comment:
```suggestion
set -o pipefail
set -x
```

We may want to see debug logs. Is there any concerns to increase log level?

Reply:
Done